src/utils/object.ts

```diff
@@ -53,3 +53,23 @@ export function startsWith(target: string, searchString: string, position?: numb
 
   return target.indexOf(searchString, effectivePosition) === effectivePosition;
 }
+
+/**
+ * Returns true if the searchString sequence is the same as the corresponding target sequence
+ * starting at the given target position (defaulting to 0); otherwise returns false.
+ * @param target The string to search within.
+ * @param searchString The string to search for.
+ * @param position The position to start searching witin the target.
+ */
+export function endsWith(target: string, searchString: string, position?: number): boolean {
```

Comment:
Since we're attempting to match ES2015 `String.prototype.endsWith` behavior, how would you feel about adding tests for this function to [object-utils.spec.ts](https://github.com/fullstorydev/fullstory-data-layer-observer/blob/272309ffb01fa7874d5b271bdb34eac09bc0ef99/test/object-utils.spec.ts) to verify the behavior matches? Tests for our local `startsWith` implementation should provide a good pattern.